manifest.json

```diff
@@ -1,6 +1,21 @@
 {
-    "name": "Hello, world!",
+    "name": "Totally for real actual ghostery extension",
     "version":"0.000001",
```

Comment:
kek

Reply:
Let's convert this to the less droll but more semver-compliant version of this: `0.0.1`.